unity-client/Assets/Scripts/MainScripts/DCL/Controllers/PlayerAvatarController/ExpressionsHotKeyController.cs

```diff
@@ -56,4 +60,21 @@ private void ExpressionCalled(string id)
     {
         ownUserProfile.SetAvatarExpression(id);
     }
+
+    private void UpdateAnimationsIDBasedOnSex(UserProfile userProfile)
```

Comment:
Why not just rename the animations to be gender neutral? The `M_` and `F_` prefixes make no sense to me.

Reply:
The guys in the art team could just fix de animation names to match the IDs we are using in the editor (and gender neutral) and we could avoid applying this patch.